test/unit/session/Business/Service/SessionCookieServiceTest.php

```diff
@@ -204,21 +208,19 @@ private function createSessionCookieAttributeFactoryMock(): SessionCookieAttribu
         private function createSessionCookieAttributeCollection(): SessionCookieAttributeCollection
         {
             return (new SessionCookieAttributeCollection())
-                ->add(new SessionCookieAttribute(self::SESSION_ATTRIBUTE_NAME, self::SESSION_ATTRIBUTE_VALUE));
+                ->add(new SessionCookieAttribute(self::SESSION_ATTRIBUTE_NAME, self::SESSION_ATTRIBUTE_VALUE))
+                ->add(new SessionCookieAttribute('something', 'xyz'))
+                ->add(new SessionCookieAttribute('lifetime', '2'))
+                ->add(new SessionCookieAttribute('path', '/'));
         }
 
         private function expectCookieParametersCall(string $domain, int $lifetime): void
         {
-            self::setGlobalFunctionExpectations('session_get_cookie_params', compact('domain', 'lifetime'));
-            self::setGlobalFunctionExpectations(
-                'session_set_cookie_params',
-                true,
-                $lifetime,
-                $this->createSessionCookieAttributeString(),
-                $this->getCookieDomain($domain),
-                Request::isHttps(),
-                true
-            );
+            self::setGlobalFunctionExpectations('session_get_cookie_params', compact('lifetime', $lifetime));
+            self::setGlobalFunctionExpectations('session_get_cookie_params', compact('path', $this->createSessionCookieAttributeString()));
+            self::setGlobalFunctionExpectations('session_get_cookie_params', compact('domain', $this->getCookieDomain($domain)));
+            self::setGlobalFunctionExpectations('session_get_cookie_params', compact('secure', Request::isHttps()));
+            self::setGlobalFunctionExpectations('session_get_cookie_params', compact('httponly', true));
```

Comment:
compact support array argument if it's contains names of variable in current context
```suggestion
            self::setGlobalFunctionExpectations('session_get_cookie_params',['path' => $this->createSessionCookieAttributeString()]);
            self::setGlobalFunctionExpectations('session_get_cookie_params', ['domain' => $this->getCookieDomain($domain)]);
            self::setGlobalFunctionExpectations('session_get_cookie_params',['secure'=> Request::isHttps()]);
            self::setGlobalFunctionExpectations('session_get_cookie_params', ['httponly'=> true]);
```